.github/actions/kibana-ftr/action.yml

```diff
@@ -0,0 +1,79 @@
+name: 'Run Kibana FTR'
+description: 'Run Kibana tests using FTR'
+
+inputs:
+  test_kibana_url:
+    description: 'URL for the Kibana instance to test'
+    required: true
+  test_es_url:
+    description: 'URL for the ES instance'
+    required: true
+  es_version:
+    description: 'Version of Elasticsearch to test against'
+    required: true
+  ref_commit_sha:
+    description: 'Kibana PR commit sha'
+    required: false
+    default: 'main'
+  test_browser_headless:
+    description: 'Whether to run tests in headless mode'
+    required: false
+    default: '1'
+  test_cloud:
+    description: 'Whether to enable cloud testing'
+    required: false
+    default: '1'
+  es_security_enabled:
```

Comment:
could we remove this and go with a default of `true` ? is there a use case for running it with `false` ?

Reply:
Removed from the input and set to '1' in the action step.